frontend/src/components/activity/content/columnLayout/calculateNextSlotName.js

```diff
@@ -2,7 +2,7 @@ import cloneDeep from 'lodash/cloneDeep'
 
 export function calculateNextSlotName(slotNames) {
   let i = 0
-  // eslint-disable-next-line no-constant-condition
+
```

Comment:
was this rule dropped?

Reply:
no, but maybe it is more sophisticated and detects the if statement in the loop.